web/src/style-helpers.ts

```diff
@@ -12,6 +12,7 @@ export enum Color {
   white = "#ffffff",
 
   offWhite = "#eef1f1",
+  gray7 = "#CCDADE",
```

Comment:
Competing color naming schemes, commence!

Reply:
hahaha, Surbhi and I had a convo on this yesterday...i think we should throw out the old UX first, then go back and reconcile all the color scales.